### PR TITLE
Update install_icons_and_menus.sh

### PR DESCRIPTION
**Pull request by alexanderkotsev**
Added icon for INSPIRE resouces. I use the opportunity to confirm that my contributions to OSGeo-Live will be compatible with the OSGeo-Live license guidelines at the time of contribution. I'd like to contribute to the inclusion of INSPIRE-related resources to OSGeo-Live.

### CODE REVIEW

File: bin/install_icons_and_menus.sh [hunk 1]
```diff
@@ -462,6 +462,23 @@ EOF
 cp -a "/usr/share/applications/$WORKSHOP_INSTALL_FILE" "$USER_HOME/Desktop/"
 chown $USER_NAME.$USER_NAME "$USER_HOME/Desktop/$WORKSHOP_INSTALL_FILE"
 
+##### Setup INSPIRE installation icon
+WORKSHOP_INSTALL_FILE="inspire.desktop"
```
Comment (kalxas) [inline bin/install_icons_and_menus.sh]: This should be named INSPIRE_INSTALL_FILE

File: bin/install_icons_and_menus.sh [hunk 2]
```diff
@@ -462,6 +462,23 @@ EOF
...
+cat << EOF > "/usr/share/applications/$INSPIRE_INSTALL_FILE"
+[Desktop Entry]
+Name=INSPIRE resources
+Comment=Resources for implementation of the EU INSPIRE Directive
+Exec=firefox https://wiki.osgeo.org/wiki/INSPIRE
```
Comment (kalxas) [inline bin/install_icons_and_menus.sh]: Lets make sure this is reserved for us.

File: bin/install_icons_and_menus.sh (hunk 1 above)
Comment (alexanderkotsev) [reply bin/install_icons_and_menus.sh]: Indeed so. Thanks.

**Comment by kalxas** [comment]
Thank you @alexanderkotsev for your contribution.
Welcome to the team.
Can you please add your name in this file via a Pull Request?
https://github.com/OSGeo/OSGeoLive-doc/blob/master/contributors.csv

File: bin/install_icons_and_menus.sh (hunk 1 above)
Comment (kalxas) [reply bin/install_icons_and_menus.sh]: I will commit this fix after we merge

**Pull request merged**